src/main/java/edu/neu/EDE/io/WorkbookReader.java

```diff
@@ -0,0 +1,170 @@
+        }
+        return rowIndex - 1;
+    }
+
+
+    void extractLookZoneDataFromSheet(XSSFSheet sheet, int startLookZone) {
+        List<Integer> nullIndices = getNullRowIndices(sheet, startLookZone);
+        int numStats = getNumStats(sheet, nullIndices);
+        readLookZoneData(sheet, numStats, nullIndices);
+    }
+
+    List<Integer> getNullRowIndices(XSSFSheet sheet, int startIndex) {
+        List<Integer> nullIndices = new ArrayList<Integer>();
+        int rowCounter = startIndex;
+        Row row;
+        while (true) {
+            row = sheet.getRow(rowCounter++);
+            if (row == null) {
+                nullIndices.add(rowCounter - 1);
+                if (sheet.getRow(rowCounter) == null) {
+                    break;
+                }
+            }
+        }
+        // we need to increment the first null row by 1 since there is an extra text field to ignore
+        nullIndices.set(0, nullIndices.get(0) + 1);
+        return nullIndices;
+    }
+
+    int getNumStats(XSSFSheet sheet, List<Integer> nullIndices) {
+        // get the last not null row
+        int rowIndex = nullIndices.get(nullIndices.size() - 1) - 1;
+        int numStats = 0;
+        while (sheet.getRow(rowIndex--).getCell(0) != null) { // loop until no more stats
+            numStats++;
+        }
+        return numStats;
+    }
+
+    void readLookZoneData(XSSFSheet sheet, int numStats, List<Integer> nullIndices) {
+        int numLookZones = nullIndices.size() - 1; // last entry is null line after final lookZone
+        // getting first lookZone (may be out)
+        String stimulus, statName;
+        Double statValue;
+        Row row;
+        for (int lookZoneIndex = 1; lookZoneIndex <= numLookZones; lookZoneIndex++) {
+            if (numLookZones > lookZoneIndex) {
+                stimulus = getStimulusName(sheet, nullIndices, lookZoneIndex);
+            } else {
+                stimulus = media + LOOK_ZONE_OUT_SUFFIX;
+            }
+            int endIndex = nullIndices.get(lookZoneIndex) - 1;
+            int startIndex = endIndex - numStats;
+            addLookZoneData(sheet, startIndex, endIndex);
+        }
+    }
+
+    void addLookZoneData(XSSFSheet sheet, int startIndex, int endIndex) {
+        Row row;
+        String statName;
+        Double statValue;
+        for (int i = endIndex; i > startIndex; i--) {
+            row = sheet.getRow(i);
+            Cell statNameCell = row.getCell(STATISTIC_NAME_CELL);
+            statName = statNameCell.getStringCellValue();
+            Cell statValueCell = row.getCell(STATISTIC_VALUE_CELL);
+            statValue = statValueCell == null ? null : statValueCell.getNumericCellValue();
+            // lookZoneData.set(subject, media, stimulus, statName, statValue);
+        }
+    }
+
+    String getStimulusName(XSSFSheet sheet, List<Integer> nullIndices, int lookZoneIndex) {
+        // TODO: lookZoneIndex may not be correct; look zones can be out of order
```

Comment:
look zones aren't necessarily in numerical order.  I'm leaving this in as a known bug for now and will write up an issue to parse the lookzone number from the name cell